app/assets/yge/css/main.css

```diff
@@ -22,7 +22,7 @@ body {
   line-height: 1;
   font-family: futura-pt, sans-serif !important;
   -webkit-text-size-adjust: none;
-  overflow-x: hidden;
+  overflow: hidden;
```

Comment:
This will actually change the overflow in all the pages and prevent scrolling all other pages according to me. Can you please check?

Reply:
![ezgif-4-ccb4d55776](https://user-images.githubusercontent.com/1824604/41975740-912317aa-79e9-11e8-9ee5-a77c45d68248.gif)

GIF showing scrolling on home page and some other pages with the changes. Let me know if there's any page in particular you're worried about.

Reply:
@premachb well it happens in the url `/gas` but I guess we need to make design modifications in there to fit it without scroll because the unnecessary scroll because of the background animation also takes place.